### ISSUE TITLE
Import Order Example is not running according to properties

### ISSUE THREAD

**Issue by Ritik2604** [checkstyle/checkstyle#10000]
Bug is detected at #5982

```
$ cat config.xml
<!DOCTYPE module PUBLIC
        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
        "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
  <module name="TreeWalker">
    <property name="tabWidth" value="7"/>
     <module name="ImportOrder">
       <property name="groups" value="java, javax"/>
       <property name="option" value="top"/>
     </module>
  </module>
</module>

$ cat Test.java
import javax.io.File.*;
import static java.io.File.listRoots;
import static javax.swing.WindowConstants.*;
import static javax.swing.WindowConstants.*;
import static com.puppycrawl.tools.checkstyle.*;


public class Test{
	
}
```
```
$ java -jar checkstyle-8.41-all.jar -c config.xml Test.java

Starting audit...
[ERROR] C:\Users\nares\Downloads\checkstyle\Test.java:2:1: Wrong order for 'com.puppycrawl.tools.checkstyle.*' import. [ImportOrder]
Audit done.
Checkstyle ends with 1 errors.
```
 there should not be any error as `import` is at top compare to ` static imports`
and when I keep the `import` at below the `static imports` there is no violation 

![Capture](https://user-images.githubusercontent.com/49342895/118727354-3fab0880-b850-11eb-8c20-80ba8bf62277.PNG)

User using `staticGroups` or `useContainerOrderingForStatic`  or having only static imports then `option` property` top or bottom` has no significance If we think logically.

**Comment by Ritik2604** [comment]
@romani approve this.

**Comment by Ritik2604** [comment]
@romani Approve it and add label Bug to it.
